src/djangosaml2_spid/identity_providers.py

```diff
@@ -0,0 +1,17 @@
+# source: https://registry.spid.gov.it/identity-providers
+
+import urllib.request
+import json
+
+identity_providers_url = 'https://registry.spid.gov.it/assets/data/idp.json'
```

Comment:
that's something that would be configured in settings and not hardcoded here if you agree

Reply:
Yes I agree with you; this file is just the first prototype I made, so it surely needs many improvements.
On in! 👍🏻

Reply:
I wrote a first version of the utility and added instruction to the README.md, check it out! 😄